src/components/Board.js

```diff
@@ -16,18 +16,99 @@ class Board extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const BOARD_URL =  `https://inspiration-board.herokuapp.com/boards/${this.props.boardName}/cards`;
```

Comment:
Consider bringing up the beginning part of the `BOARD_URL` into a different constant, so you can do something like

```
const BOARD_BASE_URL = `https://inspiration-board.herokuapp.com/boards/`;

...

componentDidMount = () => {
  const BOARD_URL = `${this.BOARD_BASE_URL} ${this.props.boardName}/cards`;
...
```